pkgs/development/python-modules/sebaubuntu-libs/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  gitpython,
+  poetry-core,
+  pyelftools,
+  requests,
+}:
+
+buildPythonPackage {
+  pname = "sebaubuntu-libs";
+  version = "1.5.0-unstable-2025-07-25";
```

Comment:
I don't think it's necessary to adopt the unstable version. There are no significant changes compared to v1.5.0.
https://github.com/sebaubuntu-python/sebaubuntu_libs/compare/v1.5.0...d71307e3bf57d2941e729aa5aa67d7d37b217d89